src/bindings/python/tests/test_utils/test_utils.py

```diff
@@ -94,3 +97,94 @@ def mock_walk(path):
     result = get_cmake_path()
 
     assert result == ""
+
+
+class Maker:
+    def __init__(self):
+        self.calls_count = 0
+
+    def __call__(self, tensor: ov.Tensor) -> None:
+        self.calls_count += 1
+        tensor_data = np.array([2, 2, 2, 2], dtype=np.float32).reshape(1, 1, 2, 2)
+        ov.Tensor(tensor_data).copy_to(tensor)
+
+    def called_times(self):
+        return self.calls_count
+
+
+def create_model(maker):
+    input_shape = ov.Shape([1, 2, 1, 2])
+    param_node = ov.opset13.parameter(input_shape, ov.Type.f32, name="data")
+
+    postponned_constant = make_postponed_constant(ov.Type.f32, input_shape, maker)
+
+    add_1 = ov.opset13.add(param_node, postponned_constant)
+
+    const_2 = ov.op.Constant(ov.Type.f32, input_shape, [1, 2, 3, 4])
+    add_2 = ov.opset13.add(add_1, const_2)
+
+    return ov.Model(add_2, [param_node], "test_model")
+
+
+def test_save_postponned_constant():
+    maker = Maker()
+    model = create_model(maker)
+    assert maker.called_times() == 0
+
+    model_export_file_name = "out.xml"
+    weights_export_file_name = "out.bin"
```

Comment:
```suggestion

def test_save_postponned_constant(request, tmp_path):
    maker = Maker()
    model = create_model(maker)
    assert maker.called_times() == 0

    model_export_file_name, weights_export_file_name = create_filenames_for_ir(request.node.name, tmp_path)
```

Reply:
and same for other tests

Reply:
even better: use [this](https://github.com/akuporos/openvino/blob/master/src/bindings/python/tests/test_graph/test_manager.py#L53) as example fixture you may add

Reply:
done